src/client/ui/panels/ViewportPanelContainer.js

```diff
@@ -40,15 +41,35 @@ class ViewportPanelContainer extends Component {
           });
         }
       }
+    } else if (item.dataTransfer && item.dataTransfer.items.length) {
+      const urlItem = Array.from(item.dataTransfer.items).find(
+        item => item.kind === "string" && item.type === "text/plain"
+      );
+      if (!urlItem) return;
+      try {
+        this.props.showDialog(ProgressDialog, {
+          title: "Importing Asset",
+          message: "Importing asset..."
+        });
+        const url = (await new Promise(resolve => urlItem.getAsString(resolve))).trim();
+        const importedUri = await this.props.editor.project.import(url);
+        this.props.editor.addGLTFModelNode("Model", importedUri);
```

Comment:
Might be nice to try to name this better -- is there any metadata we can pull from the GLTF file?

Reply:
Or, if not, give it a unique name like "Model #xxxx" or something?

Reply:
Ah, I didn't think to look in the GLTF. Looks like sketchfab might put the model name in there, and Google Poly gives us a displayName in the API.